src/components/graphs/assets.js

```diff
@@ -0,0 +1,96 @@
+const createData = (data, months) => {
+  // Create the array that will be points on the graph
+  let incidentCounts = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
We should consider if there is a better way to do this. Is the number at index 0 12 months ago? Is it always January? This is ambiguous. I wonder if there is a way to label these as the months they represent in calendar order then reference them later by that name. Let's set a time to circle back to this.

Reply:
For sure, this was a quick and hacky solution, I'm sure there's a better way now that my code base is solid.

The first index will always reference the first month of the year relative to the user. So for us, since we're in Dec, the first index represents January. The first month representation changes depending on what time of year it is

Reply:
You can probably have this array populated by the same function that dynamically creates the labels in order.